include/core/CMemoryDec.h

```diff
@@ -115,12 +117,12 @@ constexpr std::size_t storageNodeOverhead(const T&) {
 
 template<typename K, typename V, typename H, typename P, typename A>
```

Comment:
I think it's worth mentioning the Boost version here, so that anyone doing a future Boost upgrade will find this and consider if it's still true.

```suggestion
//! In Boost 1.83 the bucket lists and node lists are singly linked, hence
//! only 1 pointer is needed here
template<typename K, typename V, typename H, typename P, typename A>
```

The same comment can be added to the next method too.